docker-compose-developer.yml

```diff
@@ -59,7 +58,8 @@ services:
       - ./grafana-plugin:/etc/app
       - node_modules_dev:/etc/app/node_modules
       # https://stackoverflow.com/a/60456034
-      - ${ENTERPRISE_FRONTEND:-/dev/null}:/etc/app/frontend_enterprise
+      # see the explaination above that uses the $ENTERPRISE_ENGINE env var
+      - ${ENTERPRISE_FRONTEND:-/dev/null}:${ENTERPRISE_FRONTEND_VOLUME_MOUNT_DEST_DIR:-/tmp/empty:ro}
```

Comment:
updating this because I got this locally:
```bash
Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/dev/null" to rootfs at "/etc/app/frontend_enterprise": mount /dev/null:/etc/app/frontend_enterprise (via /proc/self/fd/9), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
make: *** [start] Error 1
```

See the `oncall-private` change [here](https://github.com/grafana/oncall-private/commit/022ecb329487aa9cfa8bee66ab970588aa6c0aa9) related to this